src/lib/SipClient.js

```diff
@@ -86,8 +105,42 @@ export default class SipClient extends events.EventEmitter {
         this._ua.call(number, constraints);
     }
 
+    sendMessage(string_msg, opts = {} ) {
+        logger.debug("sendMessage() %s", string_msg);
+        if (!this._rtcSession) {
+            logger.debug("sendMessage() No rtcSession is not set");
+            return;
+        }
+
+        // We have to temporarily change the ua display_name before sending the message
```

Comment:
I'd argue the name should be set before joining the confbridge and it should then be locked in.... we can add another field to the pre call "do you look spiffing" to save people going into settings?